Makefile

```diff
@@ -392,6 +392,15 @@ launch-loki:
 clean-launch-loki:
 	helm uninstall loki
 
+launch-dex:
+	helm repo add stable https://kubernetes-charts.storage.googleapis.com/
```

Comment:
Using --force-update (as in `helm repo add --force-update stable ...`) might be good here—in a few other places (e.g. our loki tests) it's prevented CI flakes when Minikube chokes and travis retries the make target